docs/source/contribution.rst

```diff
@@ -261,6 +261,15 @@ Please note the followings when writing the document.
   original one, users should explicitly describe only what is implemented in
   the document.
 
+For changes that modify or add new Cython files, please make sure the pointer types follow these guidelines (`GH-1913 <https://github.com/cupy/cupy/issues/1913>`_).
```

Comment:
How about this? This is how we link to GitHub issues. (see https://github.com/cupy/cupy/blob/master/docs/source/upgrade.rst for example)
```suggestion
For changes that modify or add new Cython files, please make sure the pointer types follow these guidelines (`#1913 <https://github.com/cupy/cupy/issues/1913>`_).
```

Reply:
Thanks @niboshi! I knew I've seen a GH issue link somewhere!